delphi/translators/for2py/preprocessor.py

```diff
@@ -170,13 +171,40 @@ def process_includes(lines, infile):
 
     return lines
 
+
+def refactor_select_case(lines):
+    """Search for lines that are CASE statements and refactor their structure
+    such that they are always in a i:j form. This means any CASE statement that
+    is in the form <:3> will be <Inf:3>. This is done so that the FortranOFP
```

Comment:
This seems odd.  Should <:3> be transformed to <(minus infinity):3> ?

Reply:
That's correct. Since I was using `Inf` just as a placeholder to help me know when a `<=` or `>=` needs to be written and the preprocessed Fortran code with `Inf`s added to it would be invalid anyway, I didn't think about making it semantically correct. I'll add the `-Inf` right now.

Reply:
I see.  I didn't realize that Inf was just a placeholder.  In that case, it's fine to ignore my comment about "minus infinity" -- maybe just add a comment saying that Inf is a placeholder.